code/RedirectedURL.php

```diff
@@ -80,8 +80,8 @@ public function setFromQuerystring($val) {
 	}
 	
 	public function setTo($val) {
-		$val = rtrim($val,'?');
-		if($val != '/') $val = rtrim($val,'/');
+		//$val = rtrim($val,'?');
+		//if($val != '/') $val = rtrim($val,'/');
```

Comment:
edit: actually, who's to say that the URL should definitely have a trailing slash, what if the "to" url is "local-attachment.pdf"?